_events/2020/2020-11-sc20.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Research Software Engineers in HPC (RSE-HPC-2020)
+subtitle: Creating Community, Building Careers, Addressing Challenges
+location: SC20, Atlanta, GA
```

Comment:
or remote?

Reply:
Yep looks like this is stated below! I would put up here something to reflect that, here is a suggestion
```suggestion
location: SC20, Atlanta, GA or remote
```

Reply:
Oops, meant to add that.  Will fix.